src/systems/tracked-controls.js

```diff
@@ -10,7 +11,8 @@ module.exports.System = registerSystem('tracked-controls', {
 
     this.controllers = [];
 
-    this.updateControllerList();
+    this.updateControllerList(navigator.getGamepads && navigator.getGamepads());
+    this.throttledUpdateControllerList = utils.throttle(this.updateControllerList, 500, this);
```

Comment:
it might be nice to allow this throttle duration to be configured by a component value (with a default value of `500` milliseconds)